Homework2_ZhukAlexander/task3.py

```diff
@@ -0,0 +1,12 @@
+# Найти самое длинное слово в предложении
+
+import re
+newStr = input("Предложение: ")
+newStr.strip()
```

Comment:
bad naming - we use snake case according to PEP8

Reply:
done